### DISCUSSION TITLE
Who committed the crime?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1000] [Galaxy Trivia] [answered]
After a local 🏦 robbery, four suspects were interviewed. Below is a summary of their statements.

The 👮 know that each of them told the truth in one of the statements, but lied in the other.

**Ashley said:** It wasn't Drew. It wasn't Billie.
**Billie said:** It wasn't Casey. It was Drew.
**Casey said:** It was Ashley. It wasn't Drew.
**Drew said:** It was Casey. It wasn't Ashley.

**Comment by Sriansh-raj** [comment]
To solve the crime, we need to determine which statement is true and which is false for each suspect. Let's analyze each statement:
- Ashley said: It wasn't Drew. It wasn't Billie. If this statement is true, then Drew and Billie are both innocent. If this statement is false, then either Drew or Billie is guilty.
- Billie said: It wasn't Casey. It was Drew. If this statement is true, then Drew is guilty and Casey is innocent. If this statement is false, then Casey is guilty and Drew is innocent.
- Casey said: It was Ashley. It wasn't Drew. If this statement is true, then Ashley is guilty and Drew is innocent. If this statement is false, then Drew is guilty and Ashley is innocent.
- Drew said: It was Casey. It wasn't Ashley. If this statement is true, then Casey is guilty and Ashley is innocent. If this statement is false, then Ashley is guilty and Casey is innocent.

Based on the above analysis, we can conclude that Drew is the only suspect who is mentioned in two statements, and one of them must be true. Therefore, **Drew** is the guilty suspect.

**Comment by peterbaccaro** [comment]
We know that each of them told the truth in one of the statements and lied in the other.
So according to Ashley, as one of the statements is false, it must have been either Drew or Billie.
If it was Drew then Billie's statements would both be true, which isn't allowed.
Therefore it was **Billie** that robbed the bank, which agrees with Casey and Drew's statements.

**Comment by thuongtruong1009** [comment, marked as answer]
If Ashley is telling the truth in her first statement, it means that both Drew and Billie are innocent. However, if Ashley is telling the truth in her second statement, it means that Drew is guilty. Since one statement must be true and the other false, we can conclude that Ashley is lying in her first statement and telling the truth in her second statement.

If Billie is telling the truth in his statement, it means that Casey is innocent and Drew is guilty. However, if Billie is telling the truth in his second statement, it means that Drew is guilty. Again, since one statement must be true and the other false, we can conclude that Billie is lying in his first statement and telling the truth in his second statement.

If Casey is telling the truth in his statement, it means that Ashley is guilty and Drew is innocent. However, if Casey is telling the truth in his second statement, it means that Drew is guilty. Similarly, since one statement must be true and the other false, we can conclude that Casey is lying in his first statement and telling the truth in his second statement.

Lastly, if Drew is telling the truth in his statement, it means that Casey is guilty and Ashley is innocent. However, if Drew is telling the truth in his second statement, it means that Ashley is guilty. Once again, since one statement must be true and the other false, we can conclude that Drew is lying in his first statement and telling the truth in his second statement.

Based on the analysis, we can determine that Drew is the one who committed the crime, as Billie's second statement, Casey's second statement, and Ashley's second statement all point to Drew's guilt.

**Comment by CelioAmaral** [comment]
After a local bank robbery, four suspects were interviewed. From the statements...

Who committed the crime?

Answer
Billie robbed the bank.

From the information we were able to say who committed the crime.

Reasoning
We know that each of them told the truth in one of the statements and lied in the other.

So according to Ashley, as one of the statements is false, it must have been either Drew or Billie.

If it was Drew then Billie's statements would both be true, which isn't allowed.

Therefore it was Billie, which agrees with Casey and Drew's statements.
